draft-ietf-tls-rfc8446bis.md

```diff
@@ -4853,15 +4853,11 @@ Security issues are discussed throughout this memo, especially in
 
 #  IANA Considerations
 
-[[OPEN ISSUE: Should we remove this? I am reluctant to create a situation
-where one needs to read 8446 to process this document.]]
-
-[[OPEN ISSUE: Add some text to rename the extended_master_secret entry in
-the extensions registry to extended_main_secret, after the above is resolved.]]
-
-
 This document uses several registries that were originally created in
-{{RFC4346}} and updated in {{?RFC8447}}. IANA has updated these to reference this document.
+{{RFC4346}} and updated in {{RFC8446}} and {{?RFC8447}}. The changes
+between {{RFC8446}} and this document are described in {{bis-changes}}.
```

Comment:
It might not matter, but shouldn't it be "changes since 8447"?

Reply:
I think both, because some of the changes were in 8446.